dali/operators/signal/fft/spectrogram_gpu.cc

```diff
@@ -82,13 +82,16 @@ struct SpectrogramOpImplGPU : public OpImplBase<GPUBackend> {
     TensorListShape<> out_shape;
     in_shape_1D.resize(in_shape.num_samples());
 
+    for (int i = 0; i < in_shape.num_samples(); i++) {
+      if (volume(in_shape.tensor_shape_span(i)) == 0) {
+        DALI_FAIL(make_string("Spectogram does not support empty (0-volume) samples. The sample ",
+                              i, " shape is ", in_shape[i]));
+      }
+    }
```

Comment:
Isn't it a breaking change? Wouldn't it have worked previously if there was at least one non-empty sample?

Reply:
It wouldn't. The test I added fails with invalid mem access, even though only the first sample is zero-vol.